app/api/contact/route.ts

```diff
@@ -0,0 +1,13 @@
+import { successResponse, errorResponse } from '~/lib/utils/apiResponse';
+import { validateContactData } from '~/lib/utils/validateContactData';
+
+export async function POST(request: Request) {
+  const data = await request.json();
+  const errors = validateContactData(data);
+
+  if (errors.length > 0) {
+    return errorResponse(errors, 400);
```

Comment:
`400` is by default, isn't it?
```suggestion
    return errorResponse(errors);
```